data/sudarshan-doiphode.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Sudarshan Doiphode",
+  "type": "personal",
+  "bio": "I am a Java Full Stack Developer who is more interstead in Devops nowadays",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/100bitdarsh",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/sudarshan-doiphode",
+      "icon": "FaGitHub"
```

Comment:
You may want to consider replacing this with **' FaGithub '** since the current icon is not valid :+1: 
you can also checkout [ICONS](https://linkfree.eddiehub.io/icons) for correct icons :rocket: 
```Suggestion
      "icon": "FaGithub"
```